README.md

```diff
@@ -16,44 +16,65 @@ ansible-galaxy collection install -vv -r meta/collection-requirements.yml
+pod and corresponding systemd unit to manage.  The format of the `dict` is
+mostly like the [podman_play
+module](https://docs.ansible.com/ansible/latest/collections/containers/podman/podman_play_module.html#ansible-collections-containers-podman-podman-play-module)
 except for the following:
 
-* `generate_systemd` - If you specify a value of `true`, then the role will
-  create a systemd unit to run the container using the role default variables.
-  Or, you can specify a `dict` in the `podman_container` module format.
-* `generate_systemd.container_prefix` is hard-coded to `lsr_container`
-* `generate_systemd.separator` is hard-coded to `-`
-* `generate_systemd.path` is hard-coded to `/etc/systemd/system` for root
-  containers, and `$HOME/.config/systemd/user` for non-root containers
-* `run_as_user` - Use this to specify a per-container user.  If you do not
+* `run_as_user` - Use this to specify a per-pod user.  If you do not
   specify this, then the global default `podman_run_as_user` value will be used.
   Otherwise, `root` will be used.  NOTE: The user must already exist - the role
   will not create.
-* `run_as_group` - Use this to specify a per-container group.  If you do not
+* `run_as_group` - Use this to specify a per-pod group.  If you do not
   specify this, then the global default `podman_run_as_group` value will be
   used.  Otherwise, `root` will be used.  NOTE: The group must already exist -
   the role will not create.
 * `systemd_unit_scope` - The scope to use for the systemd unit.  If you do not
   specify this, then the global default `podman_systemd_unit_scope` will be
   used.  Otherwise, the scope will be `system` for root containers, and `user`
   for user containers.
+* `kube_file_src` - This is the name of a file on the controller node which will
+  be copied to `kube_file` on the managed node.  This is a file in Kubernetes
+  YAML format.
+* `kube_file_content` - This is either a string in Kubernetes YAML format, or
+  a `dict` in Kubernetes YAML format.  It will be used as the contents of
+  `kube_file` on the managed node.
+* `kube_file` - If you specify either `kube_file_src` or `kube_file_content`, you
+  do not have to specify this.  It is highly recommended to omit `kube_file` and
+  instead specify either `kube_file_src` or `kube_file_content` and let the role
+  manage the file path and name.
+  * The file basename will be the `metadata.name` value from the K8s yaml, with a
+    `.yml` suffix appended to it.
+  * The directory will be `/etc/containers/kubernetes.d` for system services
+  * The directory will be `$HOME/.config/containers/kubernetes.d` for user services.
+
+For example, if you have
+```yaml
+    podman_kube_specs:
+      - state: started
+        kube_file_content:
+          apiVersion: v1
+          kind: Pod
+          metadata:
+            name: myappname
+```
+This will be copied to the file `/etc/containers/kubernetes.d/myappname.yml` on
```

Comment:
Could we rename it to `ansible-kubernetes.d`?  Just to make sure, future Podman maintainers don't create a conflict with the rather generic `kubernetes.d`.

Reply:
done